src/Microsoft.AspNet.Mvc.Core/ModelBinding/Metadata/DisplayMetadata.cs

```diff
@@ -116,6 +116,12 @@ public class DisplayMetadata
         public int Order { get; set; } = 10000;
 
         /// <summary>
+        /// Gets or sets the placeholder.
```

Comment:
Recommend stating that this is a *delegate* and not the placeholder itself. I recommend using the text from the `Description` attribute because it's the same pattern. The comment might also want to say "a delegate that gets the placeholder *text*".